README.md

```diff
@@ -71,7 +71,7 @@ export const config = {
    **/
   dataset: process.env.NEXT_PUBLIC_SANITY_DATASET || 'production',
   projectId: process.env.NEXT_PUBLIC_SANITY_PROJECT_ID,
-  apiVersion: '2021-03-25',
+  apiVersion: '2021-08-11',
```

Comment:
```suggestion
  apiVersion: '2021-10-21', // Learn more: https://www.sanity.io/docs/api-versioning
```
This would be the latest released version of the API. However, it's built so that you can just enter the current date (`YYYY-MM-DD`) and it will pick the latest version automatically.